### PR TITLE
Unify specification for `let_*` algorithms

### LINKED ISSUE #404: [editorial] the spec for all the `let_` algorithms can be combined
They are largely identical with small local edits

### PR DESCRIPTION
**Pull request by lucteo**
This fixes #404

### CODE REVIEW

File: std_execution.bs [hunk 1]
```diff
@@ -5161,197 +5161,92 @@ are all well-formed.
...
-                </pre>
-
-                Otherwise:
-
-                <pre highlight="c++">
-                completion_signatures &ltUs..., set_error_t(Ws)...>
-                </pre>
 
-    If `execution::let_stopped(s, f)` does not return a sender that invokes `f` when `set_stopped` is called, and making its completion dependent on the completion of a sender returned by `f`, and propagates the other completion-signals sent by `s`, the behavior of calling `execution::let_stopped(s, f)` is undefined.
+    If <code><i>let-cpo</i>(s, f)</code> does not return a sender that invokes `f` when <code><i>set-cpo</i></code> is called, and making its completion dependent on the completion of a sender returned by `f`, and propagates the other completion-signals sent by `s`, the behavior of calling <code><i>let-cpo</i>(s, f)</code> is undefined.
```
Comment (griwes) [inline std_execution.bs:5249]: I know this was there prior, but I think we need to s/making/makes/ to make this make sense grammatically; can you do that as a drive-by fix?

**Review by griwes — CHANGES_REQUESTED** [review]

**Comment by lucteo** [comment]
Droping this in the favour of #442. Sorry for the complicated branching here.
